### PR TITLE
Instrument accessibility note

### LINKED ISSUE #1524: Should "instrument"/"mechanism" be accessible? keyboard actionable?
Several of our rules use [instrument](https://act-rules.github.io/glossary/#instrument-to-achieve-an-objective) which is our interpretation of WCAG's [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism).

WCAG's definition has a note stating that:
> The mechanism needs to meet all success criteria for the conformance level claimed.

among other, this means meeting [2.1.1 Keyboard](https://www.w3.org/TR/WCAG21/#keyboard) (be operable by keyboard) and [4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value) (be included in the accessibility tree) which are both level A.

It seems that whether ours "instrument" need to be included and/or keyboard actionable is a recurring discussion in the related PRs (as far as I can remember). See https://github.com/act-rules/act-rules.github.io/pull/421/files#r537414878 for a recent occurrence of this discussion.

There is little doubt that in the grand scheme of things, the instrument needs to be accessible, but it is less clear that this is required by the individual SC. The note in the definition of "mechanism" is only a note…

---
So:
1. Does WCAG requires that "mechanisms" are accessible (keyboard, screen readers, …)?
2. Should our rules require that "instruments" are accessible?

The two questions are actually disjoint. We can have rules more lenient than WCAG. We can have rules with an assumption that "accessibility of the instrument is needed even if WCAG is not crystal clear on the question".

---
Rules currently using "instrument":
```bash
$ grep -l instrument _rules/*.md
_rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md
_rules/auto-play-audio-has-control-mechanism-4c31df.md
_rules/auto-update-text-efbfc7.md
_rules/device-motion-disabled-c249d5.md
_rules/device-motion-user-interface-7677a9.md
_rules/printable-characters-shortcut-ffbc54.md
```

### PR DESCRIPTION
**Pull request by Jym77**
Add a note in the definition of instrument, and the rules using it, to indicate that instruments should be accessible.

Closes issue(s):

- closes #1524 

Need for Final Call:
This will not require a Final Call (editorial changes)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: pages/glossary/instrument.md [hunk 1]
```diff
@@ -9,5 +9,12 @@ An [HTML element][] that when [activated][] allows an end-user to achieve an obj
 
 **Note**: Any rule that uses this definition must provide an unambiguous description of the objective the instrument is used to achieve.
 
+#### Background
```
Comment (WilcoFiers) [inline pages/glossary/instrument.md:12]: This is going to result in a duplicate ID, unfortunately. We don't have a fix for this yet. Maybe change this to:

```suggestion
#### Background About Instrument
```

**Review by WilcoFiers — APPROVED** [review]

**Review by WilcoFiers — DISMISSED** [review]
Wrong button.

File: pages/glossary/instrument.md [hunk 2]
```diff
@@ -9,5 +9,12 @@ An [HTML element][] that when [activated][] allows an end-user to achieve an obj
...
+
+This definition is a more restrictive version of WCAG's definition of [mechanism][], notably restricting it to the current document. WCAG has a note that "The mechanism needs to meet all success criteria for the conformance level claimed." This includes all the level A criteria such as [Success Criterion 2.1.1 Keyboard][sc211] (the mechanism must be keyboard accessible) or [Success Criterion 4.1.2 Name, Role, Value][sc412] (the mechanism must be exposed to assistive technologies and have an accessible name). This definition, and the rules using it, leaves these extra requirements out. This avoids reporting the same component twice for the same reason (e.g., missing an accessible name) under two different rules and Success Criteria, and help pinpoint the error related to each Success Criterion. Instruments should nonetheless be fully accessible at the correct conformance level (depending on the rule using them).
```
Comment (carlosapaduarte) [inline pages/glossary/instrument.md:14]:
```suggestion
This definition is a more restrictive version of WCAG's definition of [mechanism][], notably restricting it to the current document. WCAG has a note that "The mechanism needs to meet all success criteria for the conformance level claimed." This includes all the level A criteria such as [Success Criterion 2.1.1 Keyboard][sc211] (the mechanism must be keyboard accessible) or [Success Criterion 4.1.2 Name, Role, Value][sc412] (the mechanism must be exposed to assistive technologies and have an accessible name). This definition, and the rules using it, leaves these extra requirements out. This avoids reporting the same component twice for the same reason (e.g., missing an accessible name) under two different rules and Success Criteria, and helps pinpoint the error related to each Success Criterion. Instruments should nonetheless be fully accessible at the correct conformance level (depending on the rule using them).
```

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
Typo

File: _rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md [hunk 3]
```diff
@@ -72,6 +72,8 @@ The native `video` and `audio` controls in several browser and assistive technol
 
 ## Background
 
+The [instruments][instrument] used to pass this rule (if any), must meet all level A Success Criteria in order to fully satisfy [Success Criterion 1.4.2 Audio Control][sc142]. These extra requirements are left out of this rule, and of the definition of [instrument][], for the sake of simplicity.
```
Comment (WilcoFiers) [inline _rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md:75]:
```suggestion
The [instruments][instrument] used to pass this rule (if any), must meet all level A Success Criteria in order to fully satisfy [Success Criterion 1.4.2 Audio Control][sc142]. These extra requirements are left out of this rule, and should be tested separately.
```

**Review by WilcoFiers — DISMISSED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Review by ajanec01 — APPROVED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Pull request merged**